client/src/components/App/index.tsx

```diff
@@ -129,11 +130,13 @@ export const AppComponent: FunctionComponent<AppProps> = observer(props => {
       return;
     }
 
-    const streamParams = {
+    const streamParams: DataFilters = {
       namespace: store.controls.currentNamespace,
       verdict: store.route.verdict,
       httpStatus: store.route.httpStatus,
       filters: store.route.flowFilters,
+      showHost: store.controls.showHost,
```

Comment:
yeah, `showHost` is ok name for `store.controls` which is connected to how we control data transfer from user perspective, but not for `DataFilters`